src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTagHelperResolver.cs

```diff
@@ -75,32 +74,21 @@ public DefaultTagHelperResolver(ErrorReporter errorReporter, Workspace workspace
                         nameof(GetTagHelpersAsync)),
                     exception);
             }
-        }
 
-        public override TagHelperResolutionResult GetTagHelpers(Compilation compilation)
-        {
-            var descriptors = new List<TagHelperDescriptor>();
-
-            var providers = new ITagHelperDescriptorProvider[]
-            {
-                new DefaultTagHelperDescriptorProvider() { DesignTime = true, },
-                new ViewComponentTagHelperDescriptorProvider(),
-            };
-
-            var results = new List<TagHelperDescriptor>();
-            var context = TagHelperDescriptorProviderContext.Create(results);
-            context.SetCompilation(compilation);
-
-            for (var i = 0; i < providers.Length; i++)
+            if (result == null)
```

Comment:
I think we can remove the error logging here. This was important when WTE was calling this API. Now we're the only caller.